### PR TITLE
Cleanup newlines and blanks in "FormCanvas" packages

### LINKED ISSUE #10478: Cleanup newlines and blanks in "FormCanvas" packages
see #10396 and fix it for all "FormCanvas" related packages:

```Smalltalk
badCases := SystemNavigation default allMethodsSelect: [ :each | each sourceCode trimRight size ~= each sourceCode size ].

fixInPackage := badCases select: [:each | each package name beginsWith: 'FormCanvas' ].

fixInPackage do: [:each |
	| refactoring classOrTrait |
	classOrTrait := each traitSource ifNil: [ each methodClass ] ifNotNil: [ each traitSource innerClass ].
	refactoring := RBAddMethodChange compile: each sourceCode trimRight in: classOrTrait for: nil.				 
	refactoring execute	
]
```

### PR DESCRIPTION
**Pull request by astares**
Fix #10478

### CODE REVIEW

File: src/FormCanvas-Core/Bezier2Segment.class.st [hunk 1]
```diff
@@ -389,7 +387,6 @@ Bezier2Segment >> printOn: aStream [
 Bezier2Segment >> roundTo: quantum [
 	super roundTo: quantum.
 	via := via roundTo: quantum.
```
Comment (kasperosterbye) [inline src/FormCanvas-Core/Bezier2Segment.class.st:389]: Dot at end of last statement :-)

File: src/FormCanvas-Core/Bezier3Segment.class.st [hunk 2]
```diff
@@ -188,7 +187,6 @@ Bezier3Segment >> asBezier2Points: error [
 		prev := pts at: index.
 		].
 	^ pts.
```
Comment (kasperosterbye) [inline src/FormCanvas-Core/Bezier3Segment.class.st:189]: Ditto

**Review by kasperosterbye — APPROVED** [review]
The only issue is a some methods with dots at the end (I did not hand pick them all). 
Perhaps the right thing is to raise new issue with "remove dot at end" and "space before dots at end of line".

But to get on right now, I suggest to merge.

**Comment by astares** [comment]
@kasperosterbye 
Yes - hope to be able to care for the "old dots" later on. Ideally at least for new code should not introduce new ones.
It requires more work and attention (we need settings, editor change on accept, a refactoring and lint rules with autofix, ...)

File: src/FormCanvas-Core/Bezier2Segment.class.st (hunk 1 above)
Comment (StephanEggermont) [reply src/FormCanvas-Core/Bezier2Segment.class.st:389]: There are lots of extra dots. Let's not fix them in the same PR

**Review by StephanEggermont — APPROVED** [review]
Ready to go. Dot fixes should be in a separate PR

**Comment by kasperosterbye** [comment]
The extra dots - perhaps remember to also look for the situation with space before the dot? I see this one `foo   .` several places.

**Comment by astares** [comment]
@kasperosterbye Will do - have scripts to find out "unclean" situation that should later go into a release test to keep the quality level up

**Pull request merged**